packages/iot-app-kit-visualizations/src/components/charts/sc-tooltip/sc-tooltip-rows.tsx

```diff
@@ -104,15 +103,13 @@ export class ScTooltipRows {
    * i.e. if `supportsString` is false, do not return any data streams of string type
    */
   visualizedDataStreams = (): DataStream[] => {
-    const streams = this.dataStreams.filter(isSupportedDataType(this.supportString));
```

Comment:
i'm unsure if we can remove this one - status timeline uses `supportsString` to display the string datastreams in the tooltip